README.md

```diff
@@ -29,12 +29,41 @@ A third party security audit was performed by Cure53, you can see the full repor
 
 `runc` only supports Linux. It must be built with Go version 1.21 or higher.
 
+### Pre-Requisites
+
+#### Go
+
 NOTE: if building with Go 1.22.x, make sure to use 1.22.4 or a later version
 (see [issue #4233](https://github.com/opencontainers/runc/issues/4233) for
 more details).
 
-In order to enable seccomp support you will need to install `libseccomp` on your platform.
-> e.g. `libseccomp-devel` for CentOS, or `libseccomp-dev` for Ubuntu
+#### Utilities and Libraries
+
+In addition to Go, building `runc` requires multiple utilities and libraries to be installed on your system.
+
+On Ubuntu/Debian, you can install the required dependencies with:
+
+```bash
+apt update && apt install -y make gcc linux-libc-dev libseccomp-dev pkg-config git
+```
+
+On CentOS/Fedora, you can install the required dependencies with:
+
+```bash
+yum install -y make gcc kernel-headers libseccomp-devel pkg-config git
```

Comment:
Just to be extra-careful, kernel-headers is a meta-package that refers to the headers of the latest kernel in fedora and centos?

Reply:
As far as I know, yes. But I won't claim to be super expert in any package manager (well, I did once write an implementation for managing alpine apk packages in golang, which required figuring out how it did everything, but that was for a particular project in the past 😁 )

Reply:
But I mean, this package exists and these instructions work fine on the fedora container you tested, right?

Reply:
re: https://github.com/opencontainers/runc/pull/4353#issuecomment-2244980919

Awesome, thanks!